### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by stellakunzang**
deleted unused routes with the help of `traceroute` gem
includes sad path test we collaborated on earlier

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,6 +28,7 @@ gem 'acts-as-taggable-on', '~> 6.0'
 gem 'omniauth-census', git: "https://github.com/turingschool-projects/omniauth-census"
 gem 'omniauth-github', github: 'omniauth/omniauth-github', branch: 'master'
 gem 'sendgrid-ruby'
+gem 'traceroute'
```
Comment (stellakunzang) [inline Gemfile:31]: super simple. Just ran `rake traceroute` and saw unused routes as well as a big ass list of unreachable action methods, which were almost entirely application_controller methods that were unnecessarily inherited everywhere.

File: app/controllers/user_videos_controller.rb [hunk 2]
```diff
@@ -1,5 +1,4 @@
 class UserVideosController < ApplicationController
-  def new; end
```
Comment (stellakunzang) [inline app/controllers/user_videos_controller.rb:2]: this method didn't even have a route

File: Gemfile (hunk 1 above)
Comment (perryr16) [reply Gemfile:31]: Nice! that probably will increase our spec coverage

File: config/routes.rb [hunk 3]
```diff
@@ -36,21 +36,16 @@
...
   get '/invite', to: 'invites#new', as: :invite
   resources :invites, only: [:create]
 
-  resources :tutorials, only: [:show, :index] do
-    resources :videos, only: [:show, :index] 
-  end
+  resources :tutorials, only: [:show]
 
-  resources :user_videos, only:[:create, :destroy]
+  resources :user_videos, only:[:create]
```
Comment (perryr16) [inline config/routes.rb:50]: loving this gem

**Pull request merged**